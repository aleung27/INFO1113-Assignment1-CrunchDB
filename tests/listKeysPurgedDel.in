LIST KEYS
SET a 3
LIST KEYS
PURGE a
LIST KEYS
Set b 9
LIST KEYS
DEL b
LIST KEYS
