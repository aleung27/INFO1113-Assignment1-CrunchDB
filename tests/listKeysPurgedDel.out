> no keys

> ok

> a

> ok

> no keys

> ok

> b

> ok

> no keys

